020-LinuxFundamentals/Student/Challenge-03.md

```diff
@@ -0,0 +1,47 @@
+# Challenge 03 - Handling files
+
+[< Previous Challenge](./Challenge-02.md) - **[Home](../README.md)** - [Next Challenge >](./Challenge-04.md)
+
+## Description
+
+In this challenge you will learn basic commands about file manipulation such as create, rename, find and remove files.
+
+- Find in the `/var` directory all the files that have been modified in the last 60 minutes
+- Show the type of file of `/bin/htop`, `/etc/passwd` and `/usr/bin/passwd`
+- Go to your home directory and download [azure-linux.svg](https://docs.microsoft.com/en-us/learn/achievements/azure-linux.svg)  and [azure-ops-guide.pdf](https://docsmsftpdfs.blob.core.windows.net/guides/azure/azure-ops-guide.pdf) 
+- Display the type of file of `azure-linux.svg` and `azure-ops-guide.pdf`
+- Rename `azure-linux.svg` to `azure-linux.pdf` 
+- Display the type of file of `azure-linux.pdf` and `azure-ops-guide.pdf`
+- Create a directory `~/lab` and enter it.
+- Create the file `today.log` and the file `yesterday.log` in lab.
+- Check the creation date and time
+- Change the date on `yesterday.log` to match yesterday's date
+- Check the creation date and time again
+- Create a directory `~/mybackup` and copy all files from `~/lab` into it
+- Use one command to remove the directory `~/mybackup` and all files into it
+- Create a directory `~/logbackup` and copy the `*.log` files from `/var/log` into it
+- Count the number of linux from the file `/etc/wgetrc`
```

Comment:
Consider changing to "Count the number of times 'linux' appear in the file"

Reply:
Sorry had a typo in my text. Should be "Count the number of times 'linux' appears in the file"